server/src/main/java/org/elasticsearch/health/HealthService.java

```diff
@@ -35,11 +35,11 @@
 public class HealthService {
 
     // Visible for testing
-    static final String UNKNOWN_RESULT_SUMMARY_PREFLIGHT_FAILED = "Could not determine indicator state. Cluster state is not stable. Check "
-        + "details for critical issues keeping this indicator from running.";
+    static final String UNKNOWN_RESULT_SUMMARY_PREFLIGHT_FAILED = "Could not determine health status. Check details on critical issues "
+        + "preventing the health status from reporting.";
     static final String UNKNOWN_RESULT_SUMMARY_NOT_RECOVERED =
-        "Could not determine indicator state. The current node handling the health request is not ready to assess the health of the "
-            + "cluster. Try again later or execute the health API against a different node.";
+        "Could not determine health status. This node is not ready to assess the cluster health. Try again later or run the health API "
```

Comment:
Same comment as above. We should specify for which indicator we cannot determine the status.

e.g.: "Could not determine shard allocation health status..."

Also, we should say which node is not ready to asses the cluster health. Perhaps this phrase can be further simplified. e.g.: "Node {node_ID} cannot assess the cluster health."

Reply:
We discussed off-issue and decided to move this to the next edit PR.